spec/features/proposal/collection_draft_proposals/collection_draft_proposal_update_spec.rb

```diff
@@ -27,14 +27,37 @@
     end
 
     context 'when a proposal is not "in work"' do
-      before do
-        mock_submit(CollectionDraftProposal.first)
-        visit edit_collection_draft_proposal_path(@collection_draft_proposal)
+      context 'when trying to submit a proposal that is not "in work"' do
+        before do
+          mock_submit(CollectionDraftProposal.first)
+          visit edit_collection_draft_proposal_path(@collection_draft_proposal)
+        end
+
+        it 'cannot be edited' do
+          expect(page).to have_content('Only proposals in an "In Work" status can be edited.')
+          expect(page).to have_link('Cancel Proposal Submission')
+        end
       end
 
-      it 'cannot be edited' do
-        expect(page).to have_content('Only proposals in an "In Work" status can be edited.')
-        expect(page).to have_link('Cancel Proposal Submission')
+      context 'when trying to save updated data to a proposal that is not "in work"' do
+        # this context simulates the situation where a user has accessed the proposal and is editing data (typically using browser 'back')
+        # despite the proposal having already been submitted - proposal_status != 'in_work'
+        before do
+          visit edit_collection_draft_proposal_path(@collection_draft_proposal)
+          mock_submit(CollectionDraftProposal.first)
+          fill_in 'Short Name', with: 'a short name that will not be updated'
+          fill_in 'Abstract', with: 'the short name and abstract will not be updated because the satus of this proposal is not "in_work"'
+          within '.nav-top' do
+            click_on 'Done'
+          end
+        end
+
+        it 'is not updating' do
+          expect(page).to have_content('Only proposals in an "In Work" status can be edited.')
+          expect(page).to have_link('Cancel Proposal Submission')
+          expect(page).to have_no_content('a short name that will not be updated')
```

Comment:
I think I would like to see an expectation for what the short name/abstract were.

Reply:
Re: 1st comment=> It appears that the short name actually changes every time you run the spec. the abstract will remain the same but the short name changes each time (ie: 1st run: 'Éomer_4664602497', 2nd: 'Gimli_5399518471'). 
Re 2nd comment=> done.

Reply:
They are obtainable from the proposal.